aws-push-notifications-pinpoint/src/main/java/com/amplifyframework/pushnotifications/pinpoint/AWSPinpointPushNotificationsPlugin.kt

```diff
@@ -234,10 +232,13 @@ class AWSPinpointPushNotificationsPlugin : PushNotificationsPlugin<PinpointClien
         onError: Consumer<PushNotificationsException>
     ) {
         try {
+            val pinpointPayload = PinpointNotificationPayload.fromNotificationPayload(payload)
```

Comment:
need to handle nullability case here once introduced.

Reply:
This is going to be difficult since we would then have to return to customer a result that says we werent able to handle the payload.